santosfamilyfoundation/RuinaMain
# Add  toast message to notify user that spreadsheet downloaded

# Description

The current state of the app allows users to import new questions in the spreadsheet, but the user receives no indication of the download status. Now a toast will appear after the user decides to import the questions. If the download was successful, the user will see a green success message. If the download failed, the user will see a red fail message.

# Testing

1. Open the app.
2. Click "Import Questions"
3. Type in random characters
4. Click "Import"
5. See that a red message appears
6. Navigate to spreadsheet and get sharing link.
7. Open the app
8. Click "Import Questions"
9. Paste in sharing link
10. Click "Import"
11. See that a green message appears
